pool/store/mariadb_store.py

```diff
@@ -39,15 +39,17 @@ class MariadbPoolStore(AbstractPoolStore):
     Pool store based on MariaDB.
     """
     async def connect(self):
-        self.pool = await aiomysql.create_pool(
-        minsize=1, 
-        maxsize=12,
-        host=db_host,
-        port=db_port,
-        user=db_user,
-        password=db_password,
-        db=db_name )
-        
+        try:
+            self.pool = await aiomysql.create_pool(
+            minsize=1, 
+            maxsize=12,
+            host=db_host,
+            port=db_port,
+            user=db_user,
+            password=db_password,
+            db=db_name )
```

Comment:
extra space between `db_name` and `)` violates pep8, also it's a good practice to end the last argument of a multi-argument multi-line call with a `,` so that if you add a next argument one day, the patch is a clean one-line addition rather than a delete of a line, adding that line back with a `,` at the end and then adding a new line. Makes future reviews easier. In such case you'd put the closing parenthesis in a separate line.

Reply:
Fixed. Sorry about that